docker-compose.dev.yml

```diff
@@ -55,6 +55,7 @@ services:
       - S3_ACCESS_KEY=fake
       - S3_SECRET_KEY=fake
       - S3_REGION=us-east-1
+      - SLACK_HOOK=${SLACK_HOOK}
```

Comment:
if I understand this correctly, this is the admin org slack hook?
If so I think it shouldn't be initialized in the DB initialization script + validated in the config
If not, let's remove it?

Reply:
this is used only for testing purpose